src/api/useMeasureServiceApi.tsx

```diff
@@ -71,6 +71,12 @@ export class MeasureServiceApi {
           },
         }
       );
+      if (!!response && !!response.data) {
```

Comment:
```suggestion
      if (response?.data) {
```
I think the `!!` are redundant since `response` will be falsy if it's null or undefined. Also can use optional chaining to safely navigate the data structure.

Reply:
Thanks Joe! I made the change.